src/styles/utils/_mixins.scss

```
@@ -0,0 +1,6 @@
+@mixin circle ($size){
+width: $size;
+height: $size;
+border-radius: 50%;
+}
+
```

Looks like some problems with formatting:
```suggestion
@mixin circle ($size) {
  width: $size;
  height: $size;
  border-radius: 50%;
}

```

---

okay) thanks